composer.json

```diff
@@ -28,6 +29,7 @@
         "symfony/yaml": "^4.1 || ^5.1",
         "ueberdosis/html-to-prosemirror": "^1.3",
         "ueberdosis/prosemirror-to-html": "^2.2",
+        "webonyx/graphql-php": "^14.3",
```

Comment:
This is a dependency in rebing/graphql-laravel so is this necessary to also include here?

Reply:
Nope. Good find. Probably leftover from when I started out with the webonyx package on its own.